installer/tue-install-impl.bash

```diff
@@ -76,7 +76,7 @@ function tue-install-target
     tue-install-debug "Installing $target"
 
     # Check if valid target received as input
-    if [ ! -d $TUE_INSTALL_TARGETS_DIR/$target ]
+    if [ ! -d "$TUE_INSTALL_TARGETS_DIR/$target" ]
```

Comment:
Don't include `/` in quotes

Reply:
Make this change in a single commit for all the statements below so it can be reverted